src/applications/vaos/services/mocks/index.js

```diff
@@ -39,6 +39,7 @@ const confirmedV2 = require('./v2/confirmed.json');
 
 // CC Direct Scheduling mocks
 const wellHiveAppointments = require('./wellHive/appointments.json');
+const patientDetails = require('./wellHive/patientBasicReferralDetails.json');
```

Comment:
Do you think we should rename this variable & json file to referralDetails or something that more accurately describes the data it returns

Reply:
I changed this up too ^